src/dotnet/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("dotnet.Tests")]
```

Comment:
I think you need a Key here, right?

See https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.DotNet.Compiler.Common/Properties/Properties.cs#L3